src/views/studio/SignInView/SignInView.tsx

```diff
@@ -0,0 +1,54 @@
+import tileImg from '@/assets/tile-example.png'
+import videoImg from '@/assets/video-example.png'
+import { Text } from '@/shared/components'
+import React from 'react'
+import {
+  ButtonGroup,
+  CompositionWrapper,
+  Header,
+  HowItWorksButton,
+  Overlay,
+  SignInButton,
+  SignInWrapper,
+  StyledBackgroundPattern,
+  SubTitle,
+  Tile,
+  VideoImageWrapper,
+} from './SignInView.style'
+
+type SignInViewProps = {
+  onOpenModal: (val: boolean) => void
+}
+
+const SignInView: React.FC<SignInViewProps> = ({ onOpenModal }) => {
+  return (
+    <>
+      <StyledBackgroundPattern />
+      <CompositionWrapper>
+        <VideoImageWrapper>
+          <img src={videoImg} />
```

Comment:
Maybe it should be a `div` with background image? Since it doesn't have semantic value - it's part of the design. Same thing with `Tile`